Swap from XHTML to straight html5

LinkSlayer64 opened the pull request:
Also fixed validation errors, the other files may need to be edited as
well.
Up to you Remy if you wanna go this route - I personally think it's a good choice. Validation issues tick me off though ;D

LinkSlayer64 committed:
Swap from XHTML to straight html5

Also fixed validation errors, the other files may need to be edited as
well.

decause commented:
This is _awesome_

Can you confirm that you've tested it, and that it still works? If so (srsly, don't just say 'yes, it works') I'd be _thrilled_ to accept this pull request.

:sailboat:

LinkSlayer64 committed:
Additional fixes to HTML validation

Fixed the entire site (as much as I have with this local copy  so I can
sync with the upstream) validated every page with
http://validator.w3.org

LinkSlayer64 committed:
Full fix to HTML validation and formatting

So, this is the final edit for now. All files now validate properly.
The formatting of the lecture pages nearly identically matches the
original, mostly margins (and if it isn't perfect, I kept it because I
felt it looked slightly better)
Additionally I pulled from the upstream, so I have every page that is
currently on the site.

LinkSlayer64 commented:
Yes I can - I have tested every page with the W3C validator, and they all pass with only the usual warnings. We could put the validator link on it if we felt like proclaiming how we follow standards :smiley: 
This was a giant pain, but worth the effort.
For future reference - nothing but "li" should go inside "ol" or "ul" - if you have a div or a heading, it needs to go inside the "li."

decause commented:
:tophat:
